src/openepd/model/specs/generated/network_infrastructure.py

```diff
@@ -169,7 +170,26 @@ class NetworkingRacewaysV1(BaseOpenEpdHierarchicalSpec):
 class CommunicationsConduitV1(BaseOpenEpdHierarchicalSpec):
     """Tubing used to protect and route communications wiring in a building or structure."""
 
-    _EXT_VERSION = "1.0"
+    _EXT_VERSION = "1.1"
+
+    nominal_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="Nominal Diameter is also known as the mean or average outside diameter.",
+        example="100 mm",
+    )
+    outer_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the outer wall of the pipe.",
+        example="100 mm",
+    )
+    inner_diameter: LengthMmStr | None = pyd.Field(
+        default=None,
+        description="The measurement of the distance of a straight line from one point on the inner wall of the pipe.",
+        example="100 mm",
+    )
+    wall_thickness: LengthMmStr | None = pyd.Field(default=None, description="Conduit wall thickness", example="100 mm")
```

Comment:
Maybe extract to a mixin? This seems to be super-close to the above.

Reply:
Agree, will extract those properties to mixin

Reply:
Done